apps/ledger-live-desktop/src/newArch/features/MemoTag/components/MemoTagField.tsx

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import Input, { Props as InputBaseProps } from "~/renderer/components/Input";
+import Label from "~/renderer/components/Label";
+import Box from "~/renderer/components/Box";
+import { useTranslation } from "react-i18next";
+import { Flex, Text, Tooltip } from "@ledgerhq/react-ui";
+import styled from "styled-components";
+import InfoCircle from "~/renderer/icons/InfoCircle";
+
+const TooltipContainer = styled(Box)`
+  background-color: ${({ theme }) => theme.colors.palette.neutral.c100};
+  padding: 10px;
+  border-radius: 4px;
+  display: flex;
+  gap: 8px;
+`;
+
+const InstructionText = styled(Text)`
+  color: ${({ theme }) => theme.colors.primary.c80};
+  margin-top: 6px;
+  font-size: 12px;
+  line-height: normal;
+`;
+
+type MemoTagFieldProps = InputBaseProps & {
+  maxMemoLength?: number;
+  showLabel?: boolean;
+  CaracterCountComponent?: React.FC;
+  autoFocus?: boolean;
+};
+
+const MemoTagField = ({
+  warning,
+  error,
+  value,
+  onChange,
+  showLabel = true,
+  maxMemoLength,
+  CaracterCountComponent,
+  autoFocus,
+}: MemoTagFieldProps) => {
+  const { t } = useTranslation();
+  return (
+    <Box flow={1}>
+      {showLabel && (
+        <Label>
+          <Flex>
+            <span>{t("MemoTagField.label")}</span>
+            &nbsp;&nbsp;
```

Comment:
you can use `margin` instead no?

Reply:
i can for sure you right, at the same time margin between text content don't provide you a plain text rendering (example when you copy a text spaced with margin) , and that's why i preferred a native html compatible way to do that rather than margin.